EIPS/eip-nfr.md

```diff
@@ -0,0 +1,395 @@
+---
+eip: eip-nfr.md
+title: NFT Future Rewards (nFR) Standard
+description: In this EIP, we propose a multigenerational reward mechanism that rewards‌ all ‌owners of non-fungible tokens (NFT).
+author: Yale ReiSoleil <yale@iob.fi>, dRadiant (@dRadiant), D Wang, PhD <david@iob.fi>
+discussions-to: https://github.com/ethereum/EIPs/pull/5173
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-05-08
+requires: 165, 721
+---
+
+## Simple Summary
+
+This proposal outlines the interface of a new ERC721 standard extension. NFTs can then define, process, and distribute rewards based on the realized profit to former owners.
+
```

Comment:
```suggestion
```
Not a valid EIP section.  See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for the latest version of the template.